package.json

```diff
@@ -6,12 +6,12 @@
   "scripts": {
     "build": "tsc",
     "start": "yarn build && node dist/src/index.js",
-    "start:local": "ts-node-dev --transpile-only 'src/index.ts' | pino-pretty -c",
+    "start:local": "dotenv -e .env -- ts-node-dev --transpile-only src/index.ts | pino-pretty -c",
```

Comment:
what happens if the `.env` file is not present? it just doesn't set any environment variable I assume

Reply:
the flag -e is to parse the .env file, if the file does not exists, it just does not set/override the environment variables